apps/tup-ui/index.html

```diff
@@ -10,24 +10,9 @@
         baseUrl: 'https://dev.tup-services.tacc.utexas.edu',
       };
     </script>
-    <style>
-      body {
-        margin: 0px 0px 0px 0px;
-        left: 0px;
-        right: 0px;
-        top: 0px;
-        bottom: 0px;
-      }
-      html,
-      body,
-      #root,
-      #root > div {
-        height: 100%;
-      }
-    </style>
   </head>
   <body>
-    <div id="root"></div>
+    <div id="root" class="o-site"></div>
```

Comment:
ℹ️ Styles are in [`apps/tup-ui/src/pages/Login/Layout.global.css`](https://github.com/TACC/tup-ui/pull/47/files#diff-3797b2daa362806d00e00c64497a183229c685987c833099a614bc9b04b044a9).